src/styles/blocks/catalog.scss

```diff
@@ -18,8 +20,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: 300ms;
```

Comment:
don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md
![image](https://user-images.githubusercontent.com/91411692/222482105-e9f9fab6-5e3a-4e24-9e4f-c217703e1bf2.png)

```suggestion
    transition-duration: 300ms;
```